templates/search/results-contextual-en.html

```diff
@@ -14,7 +14,7 @@
 <div class="well">
 	<form class="mrgn-tp-sm">
 		<label class="wb-inv" for="sch-inp">Search Government of Canada websites</label>
-		<input class="form-control full-width" spellcheck="false" autocomplete="off" dir="ltr" id="sch-inp" name="q" type="search" data-fusion-query />
+		<input class="form-control full-width" spellcheck="false" autocomplete="off" dir="ltr" id="sch-inp" name="q" type="search" data-fusion-query aria-describedby="pers-info" />
```

Comment:
We will need to change the id and make it more specific where it self define it's context which is for "Canada.ca" and for "Search".